File: market/integration-test/java/ru/yandex/market/logistics/nesu/configuration/ClockConfiguration.java
package ru.yandex.market.logistics.nesu.configuration;

import org.springframework.context.annotation.Bean;

import ru.yandex.common.util.date.TestableClock;

public class ClockConfiguration {
    @Bean
    TestableClock clock() {
        return new TestableClock();
    }
}

File: market/integration-test/kotlin/ru/yandex/market/logistics/mqm/configuration/ClockConfiguration.kt
package ru.yandex.market.logistics.mqm.configuration

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import ru.yandex.common.util.date.TestableClock

@Configuration
class ClockConfiguration {
    @Bean
    fun clock() = TestableClock()
}
